src/client/app/translations/data.ts

```diff
@@ -225,6 +225,7 @@ const LocaleTranslationData = {
 		"help.admin.metercreate": "This page allows admins to create meters. Please visit {link} for further details and information.",
 		"help.admin.meteredit": "This page allows admins to edit meters. Please visit {link} for further details and information.",
 		"help.admin.meterview": "This page allows admins to view and edit meters. Please visit {link} for further details and information.",
+		"help.admin.unitconversionvisuals": "The \"Input Units Visual Graphic\" visually shows the units and conversions made by admins. The \"Analyzed Units Visual Graphic\" visually shows the analysis OED does on the units and conversions entered by admins.",
```

Comment:
Could you please add the standard link to the help page (see src/client/app/components/TooltipHelpComponent.tsx) to get more information (applies to all languages).